benchmarks/nel/scripts/wiki/wikidata.py

```diff
@@ -102,6 +102,12 @@ def read_entities(
                 clean_line = line.strip()
                 if clean_line.endswith(b","):
                     clean_line = clean_line[:-1]
+                to_search = clean_line.decode("utf-8").lower()
+                if "boston" in to_search and "celtics" in to_search:
+                    obj = json.loads(clean_line)
+                    obj_lbl = obj["labels"].get("en", {}).get("value", "").lower()
+                    if "celtics" in obj_lbl:
+                        x = 3
```

Comment:
What is the purpose of this section? 

Also it might be better to call `json.loads` pre-emptively if this block is entered a lot, it looks like it can happen twice for the same line.

Reply:
Even more testing code I forgot to remove. I was tracking why a [particular entity](https://www.wikidata.org/wiki/Q131371) wasn't included in the filtered dataset.